### PR TITLE
Add Constant Scalar

### PR DESCRIPTION
**Pull request by aliceice**
As per #609

### CODE REVIEW

File: src/test/java/org/cactoos/scalar/ConstantTest.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ * @checkstyle JavadocMethodCheck (500 lines)
+ */
+public final class ConstantTest {
+
+    @Test
+    public void returnsGivenValue() throws Exception {
+        MatcherAssert.assertThat(
+            "Can't return given value",
+            new Constant<>("Hello World").value(),
+            Matchers.endsWith("World")
```
Comment (amihaiemil) [inline src/test/java/org/cactoos/scalar/ConstantTest.java]: @aliceice Why not ``Matchers.equalTo("Hello World")`` here? ``endsWith`` does not guarantee that Constant doesn't change the encapsulated String somehow.

File: src/main/java/org/cactoos/scalar/Constant.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+ * <p>Contrary to {@link StickyScalar} this constant is always
+ * pre-computed.</p>
+ *
+ * <p>This class implements {@link Scalar}, which throws a checked
+ * {@link Exception}. Despite that this class does NOT throw a checked
+ * exception as it only returns a pre-computed value.</p>
+ *
+ * <p>Example:
+ *   <pre>
+ *     final Scalar&lt;String&gt constant = new Constant("Pre-Computed Value");
```
Comment (amihaiemil) [inline src/main/java/org/cactoos/scalar/Constant.java]: @aliceice If I'm not mistaken, here you should have ``new Constant<>("Pre-Computed Value")`` (missing the ``<>``). Also, I think a ``;`` is missing after ``&gt``.

**Comment by amihaiemil** [comment]
@aliceice I also saw that you opened a ticket in ``teamed/qulice`` regarding the ``@author`` regex. Trouble is, I don't think we can wait after that now (that ticket may take a long time to be resolved and PRs cannot have impediments). Some time will pass, then your PR there may get merged, then qulice has to be released etc.

I'm not sure what exactly is not supported by qulice; the dash (``-``), the dot (``.``)? Maybe just enter your github username for now?

File: src/main/java/org/cactoos/scalar/Constant.java (hunk 2 above)
Comment (aliceice) [reply src/main/java/org/cactoos/scalar/Constant.java]: You are right, I'll fix this.

**Comment by aliceice** [comment]
@amihaiemil Qulice is not recognizing the dash `-` in my surname. I'll change it to my Github username for now.

File: src/test/java/org/cactoos/scalar/ConstantTest.java (hunk 1 above)
Comment (aliceice) [reply src/test/java/org/cactoos/scalar/ConstantTest.java]: The problem here is qulice. It does not allow to use the same String twice in a class. So either I define a variable beforehand and use this in both construction and assertion, or do it like this.

What do you suggest?

Comment (amihaiemil) [reply src/test/java/org/cactoos/scalar/ConstantTest.java]: @aliceice Right, I forgot how strict Qulice is... then I would say declare a variable beforehand :)

**Comment by aliceice** [comment]
@amihaiemil I updated the PR according to your suggestions. Could you please have a look again?

**Comment by amihaiemil** [comment]
@aliceice looks good!

**Comment by amihaiemil** [comment]
@rultor good to merge

**Comment by amihaiemil** [comment]
@yegor256 ping

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was successfully finished: +15 points just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), total is [+615](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+9724](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The job [#637](https://github.com/yegor256/cactoos/issues/637) is now out of scope

**Comment by amihaiemil** [comment]
@aliceice PR is now merged. Don't forget to ask the initial ticket's reporter to close it.

**Pull request merged**